Création couches thèmes/34

ferrere-fabrice opened the pull request.

Commit:
feat: 🎸 ajout theme couche + test unitaire et integration OK

ajout des couches controller, services + ajout des tests unitaires sur
chaque couche + ajout des tests d'integrations sur get

Commit:
Merge branch 'develop' into création-couches-thèmes/34

Commit:
fix: 🐛 remove bad case

change upper case to lower case

ferrere-fabrice committed:
fix: 🐛 ajout et suppression vérification

ajout suppression inutile sur les controllers

ferrere-fabrice committed:
fix: 🐛 rename fichier et fix mauvais import

renommage des fichiers avec un (s) en trop et change du nom card service
en review service

ferrere-fabrice committed:
docs: ✏️ ajout documentation + corrige erreur message

ferrere-fabrice committed:
fix: 🐛 erreur naming route

change route theme => themes pour respecter principe restful

ferrere-fabrice committed:
fix: 🐛 fix error syntaxe et refactoring

ajout de commentaire + renommage variable mal nommé

antobrines commented on src/routes/theme.route.js:
```diff
@@ -0,0 +1,8 @@
+const themeController = require('../controllers/theme.controller');
+// const validate = require("../middlewares/validate");
+
+router.get('/:id', [], themeController.get);
```
Tu peux ne pas mettre "[ ]"

ferrere-fabrice replied:
J'ai rajouté le middleware isConnected et adapté les tests pour qu'il prennent en compte ce middleware

ferrere-fabrice committed:
refactor: 💡 A code change that neither fixes a bug or adds a fe

rajout du middleware isconnected sur route themes + ajout du token dans
les tests unitaires

BREAKING CHANGE: 🧨 3
